salt/modules/nilrt_ip.py

```diff
@@ -510,9 +513,33 @@ def get_interfaces_details():
     return {'interfaces': list(map(_get_info, _interfaces))}
 
 
-def _change_state(interface, new_state):
+def _change_state_older(interface, new_state):
```

Comment:
we should go with 'legacy' instead of 'older', as it is in other modules.

I'd rename this to something like: _change_state_legacy()